src/shrutiC/ArmstrongNum.java

```diff
@@ -0,0 +1,28 @@
+package shrutiC;
+
+public class ArmstrongNum {
+
+	void isArmstrongNum(int number) {
+		int num = 0;
+		int sumOfCubes = 0;
+
+		String string = Integer.toString(number);
+		for (int i = 0; i < string.length(); i++) {
+			num = Integer.parseInt(Character.toString(string.charAt(i)));
```

Comment:
you can use Character.getNumericValue(ch) as second option

but you here interviewer expect like you code for int number only,

![Uploading image.png…]()